tests/mock.py

```diff
@@ -1,18 +1,41 @@
 from unittest.mock import MagicMock
 from pynio import properties
 
+template = {
+    'name': 'template',
+    'properties': {
+        'name': {
+            'type': 'str',
+            'title': None,
+        },
+        'type': {
+            "type": "str",
+            "title": None
+        },
+        'value': {
+            'type': 'int',
+            'title': 'Value',
+            'default': 0
+        }
+    }
+}
 
-def mock_instance(types):
-    instance = MagicMock(spec=[
-        'config', 'blocks_types', 'droplog', 'blocks'])
+config = {
+    'name': 'name',
```

Comment:
This was set to '' for a while but not causing unit tests to fail because of

https://github.com/neutralio/pynio/pull/19/files#diff-2070fc684b39da5f02298508983af9e2R16